<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Об'єкти</string>
    </property>
    <addaction name="drawPoint"/>
    <addaction name="drawLine"/>
    <addaction name="drawRect"/>
    <addaction name="drawElipse"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Довідка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="drawPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResourse.qrc">
     <normaloff>:/img/dot.png</normaloff>:/img/dot.png</iconset>
   </property>
   <property name="text">
    <string>Крапка</string>
   </property>
  </action>
  <action name="drawLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResourse.qrc">
     <normaloff>:/img/diagonal-line.png</normaloff>:/img/diagonal-line.png</iconset>
   </property>
   <property name="text">
    <string>Лінія</string>
   </property>
  </action>
  <action name="drawRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResourse.qrc">
     <normaloff>:/img/rectangular-shape-outline.png</normaloff>:/img/rectangular-shape-outline.png</iconset>
   </property>
   <property name="text">
    <string>Прямокутник</string>
   </property>
  </action>
  <action name="drawElipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResourse.qrc">
     <normaloff>:/img/full-moon.png</normaloff>:/img/full-moon.png</iconset>
   </property>
   <property name="text">
    <string>Еліпс</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="iconResourse.qrc">
     <normaloff>:/img/add-file.png</normaloff>:/img/add-file.png</iconset>
   </property>
   <property name="text">
    <string>Нове полотно</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconResourse.qrc"/>
 </resources>
 <connections/>
</ui>
